sites/public/pages/index.tsx

```diff
@@ -22,13 +22,12 @@ export default function Home({ latestListings }) {
   const heroTitle = <>{t("welcome.title")}</>
 
   const heroInset: React.ReactNode = (
-    <div className="hero__inset">
-      <div className="hero__text">{t("welcome.heroText")}</div>
+    <div>
```

Comment:
Do we need this div now or can it be removed?

Reply:
yeah its still needed here

Reply:
What is the need you're seeing? I tried removing it locally and it still seems to work? If it's react complaining that you need a single element you can replace this with empty tags `<> </>` which won't render in the final html